backend/requirements.txt

```diff
@@ -38,9 +38,9 @@ jsonpatch==1.33
 jsonpointer==3.0.0
```

Comment:
Upgraded langchain-core from 0.3.6 to 0.3.25, which likely includes bug fixes and performance improvements.